### PR TITLE
feat(content): #2536 Logo should link to Pocket, +Feedback form

### PR DESCRIPTION
**Pull request by csadilek**
Mostly minor UI improvements. Main change is the new info icon, that displays a feedback form when hovered over, similar to Chrome:

screen shot 2017-05-08 at 5 31 23 pm

### CODE REVIEW

File: locales/en-US/strings.properties [hunk 1]
```diff
@@ -4,10 +4,10 @@ default_label_loading=Loading…
 header_top_sites=Top Sites
 header_highlights=Highlights
 # LOCALIZATION NOTE(header_stories): This is followed by (header_stories_by)
-header_stories=Trending Stories
+header_stories=Top Stories
 # LOCALIZATION NOTE(header_stories_by): This is followed by a logo of the
-# corresponding content (trending stories) provider
-header_stories_by=by
+# corresponding content (stories) provider
+header_stories_by=from
```
Comment (Mardak) [inline locales/en-US/strings.properties]: Might as well change the id to `header_stories_from` and the localization note for the previous "followed by" is no longer accurate as these two are separate now.

File: pocket-example.json [hunk 2]
```diff
@@ -2,5 +2,7 @@
   "pocket_story_endpoint": "https://getpocket.com/v3/firefox/global-recs",
   "pocket_topic_endpoint" : "https://getpocket.com/v3/firefox/trending-topics",
   "pocket_read_more_endpoint" : "https://getpocket.com/explore/trending?src=ff_new_tab",
+  "pocket_learn_more_endpoint": "http://getpocket.com/firefox_learnmore?src=ff_newtab",
```
Comment (k88hudson) [inline pocket-example.json]: `https`?

Comment (csadilek) [inline pocket-example.json]: Thank you. I'll update.

**Comment by as-pine-proxy** [comment]
[Treeherder: fb7bd8eee2930249d22b34304cb907dd274a4e78](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=fb7bd8eee2930249d22b34304cb907dd274a4e78)

**Pull request merged**